api/cloud/oc_cloud.c

```diff
@@ -45,6 +45,14 @@ cloud_reset_delayed_callback(void *cb_data, oc_trigger_t callback,
   oc_set_delayed_callback(cb_data, callback, seconds);
 }
 
+void
+cloud_reset_delayed_callback_ms(void *cb_data, oc_trigger_t callback,
+                                uint64_t milliseconds)
+{
+  oc_remove_delayed_callback(cb_data, callback);
+  oc_set_delayed_callback_ms(cb_data, callback, milliseconds);
```

Comment:
`cloud_reset_delayed_callback_ms` and oc_set_delayed_callback_ms should have the same width of the interval argument. However,`oc_set_delayed_callback_ms` takes `uint16_t` interval -> that's max 65536, so 65 seconds... that's really not much. We should change the API to at least uint32_t or uint64_t.

Reply:
`oc_set_delayed_callback_ms_v1` deprecates `oc_set_delayed_callback_ms`